src/watertap_contrib/seto/solar_models/surrogate/trough/test_trough_surrogate.py

```diff
@@ -174,7 +182,6 @@ def test_validation(self, trough_frame):
         for row in data["validation"].itertuples():
             m.fs.trough.heat_load.fix(row.heat_load)
             m.fs.trough.hours_storage.fix(row.hours_storage)
```

Comment:
Just a note for the record- after removing the usage of default ipopt and using ipopt-watertap in `test_validation`, IPOPT output shows that the search direction becomes too small and stops at 3 or 4 iterations; this happens when `hours_of_storage` is fixed to 1 but may not be limited to that (e.g., also noticed same issue when `hours_of_storage` was fixed to 2 but didn't look beyond that). However, with regular ipopt, these problematic cases solve in 1 iteration, so the problem has to do with one of the customized settings of ipopt-watertap. However, one could argue that the problem lies within the surrogate equations since solving should be seamless for both versions of ipopt. @bknueven this could be worth exploring to determine whether ipopt-watertap should undergo some adjustment (or not) considering this as a test case.